lib/pure/distros.nim

```diff
@@ -173,7 +173,7 @@ proc detectOsImpl(d: Distribution): bool =
     result = defined(haiku)
   else:
     let dd = toLowerAscii($d)
-    result = dd in toLowerAscii(uname()) or dd in toLowerAscii(release()) or dd in toLowerAscii(hostnamectl())
+    result = dd in toLowerAscii(uname()) or dd in toLowerAscii(release()) or dd in toLowerAscii("Operating System: " & hostnamectl())
```

Comment:
This doesn't make sense. ``dd`` does not contain "Operating System".

Reply:
That's my bad, I misread the code and put the string in the wrong place, I'll fix that.